recipes/petsc/meta.yaml

```diff
@@ -0,0 +1,43 @@
+{% set build = 0 %}
+{% set version = '3.7.2' %}
+{% set md5 = '26c2ff8eaaa9e49aea063f839f5daa7e' %}
+{% set blas = os.environ.get('BLAS') or 'openblas' %}
+{% set mpi = os.environ.get('MPI') or 'mpich' %}
+{% set req = {'mpich':'>=3.2', 'openmpi':'>=1.10'} %}
+
+package:
+  name: petsc
+  version: {{version}}
+
+source:
+  fn: petsc-lite-{{version}}.tar.gz
+  url: http://ftp.mcs.anl.gov/pub/petsc/release-snapshots/petsc-lite-{{version}}.tar.gz
+  md5: {{md5}}
+
+build:
+  number: {{build}}
+  binary_relocation: true
+  detect_binary_files_with_prefix: true
+  skip: true  # [win or py3k]
+  features:
+    - blas_{{blas}}
+
+requirements:
+  build:
+    - python
+    - blas 1.* {{blas}}
+    - {{mpi}} {{req[mpi]}}
+  run:
+    - blas 1.* {{blas}}
+    - {{mpi}} {{req[mpi]}}
+
+about:
+  home: http://www.mcs.anl.gov/petsc/
+  summary: 'PETSc: Portable, Extensible Toolkit for Scientific Computation'
+  license: 'BSD'
```

Comment:
What version of BSD (e.g. `BSD 3-Clause`)? Also is there a [license file]( http://conda.pydata.org/docs/building/meta-yaml.html#license-file ) that we can include?